jmix-flowui/flowui/src/main/java/io/jmix/flowui/component/gridcolumnvisibility/JmixGridColumnVisibility.java

```diff
@@ -201,6 +202,11 @@ protected String getColumnHeaderText(DataGridColumn<?> column) {
             if (headerComponent instanceof HasText hasText) {
                 headerText = hasText.getText();
             }
+            if (Strings.isNullOrEmpty(headerText)
```

Comment:
What if we still can't get the `title` from the `header`? For example, a user set a custom component to a header? 

In the issue, the is a comment that 
> property names are used for other columns

Take a look at how the default `title` is generated for a column and let's apply the same for the `menuItem` in case we cannot obtain `title` from column.